src/AppBundle/Resources/config/routing/api/routing_api_oauth2.yml

```diff
@@ -18,28 +18,26 @@ api_oauth2_new_deck:
     methods: [POST]
     defaults:
         _controller: AppBundle:Oauth2:newDeck
-    requirements:
-        id: \d+
 
 api_oauth2_save_deck:
     path: /deck/save/{id}
-    methods: [PUT]
+    methods: [POST]

```

Comment:
It seems like something weird is happening with the [PUT] requests, I'm not quite sure what.  It's working locally, so perhaps something with the specific Symfony / PHP version.

But the [POST] request for new decks is working just fine, so I figured maybe simplest solution is to just moving everything over to [POST]?